docs/reference/elasticsearch/mapping-reference/semantic-text.md

```diff
@@ -37,6 +37,10 @@ the embedding generation, indexing, and query to use.
 [quantized](/reference/elasticsearch/mapping-reference/dense-vector.md#dense-vector-quantization)
 to `bbq_hnsw` automatically.
 
+## Default and custom endpoints
+
```

Comment:
Nit: Would add a lead-in sentence here because stacked headings are usually a no-no